modules/check_in/app/services/travel_claim/client.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+module TravelClaim
+  ##
```

Comment:
Wondering if we can have one more sub-module defined for VEIS under travel_claim?  That way when we create a client interface for claims service under the claims sub-module and all BTSSS clients under travel_claim.

Reply:
As we discussed, I think it'll be easier to maintain if we have the token endpoint in the same client/service object as the claims submission.